### ISSUE TITLE
Scroll behavior does not respond to `prefers-reduced-motion` setting

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95517]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

When the user prefers reduced motion, we should honor that and not show scrolling animation.

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-0
**Page**: review-and-submit
**Environment**: prefers-reduced-motion setting is on
**Estimated Level of Effort to fix**: Medium
**Design system component involved**: Yes
**Relevant WCAG success criteria**: [2.3.3 Animation from Interactions Level AAA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview#animation-from-interactions)

PR: https://github.com/department-of-veterans-affairs/vets-website/pull/32760

### To Reproduce:
- Using Chrome DevTools simulator or your local OS accessibility settings, turn on "reduced motion" settings. 
-  Click around the page and trigger the scroll by trying to edit an item.

#### Expected Behavior
-  There should be no scrolling animation. It should just "jump" to that part of the screen.

#### Actual Behavior
-  Scroll behavior still occurs.

## Tasks
- [ ] When the user prefers-reduced-motion do not show the scroll animation.
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Scrolling animation does not show.
- [ ] Accessibility Audit issue tracking doc is updated with ticket status

**Comment by bacitracin** [comment]
Merged! Video demo if needed for sprint review next week - https://www.loom.com/share/c588bdc8f013492b81fbf60fa77dcb38?sid=c46038b4-2bdc-48d1-9b7b-bdaf523f5b59
